components/navigation/EventPostItem.js

```diff
@@ -0,0 +1,76 @@
+import { ArrowRightIcon } from '@heroicons/react/outline';
+import React from 'react';
+import moment from 'moment';
+import IconCalendar from '../icons/Calendar';
+import Community from '../icons/Community';
+import Conference from '../icons/Conference';
+import Webinar from '../icons/Webinar';
+import Heading from '../typography/Heading';
+
+function EventPostItem({ post, className, id }) {
+  const localTime = moment().format('YYYY-MM-DD'); // store localTime
+  const currentDate = localTime + 'T00:00:00.000Z';
+  const title = post.title || '';
+  let color = '';
+  let icon = '';
+  let type = '';
+  if (title.includes('community')) {
+    icon = <Community />;
+    color = 'text-green-800';
+    type = 'COMMUNITY';
+  } else if (title.includes('conference')) {
+    icon = <Conference />;
+    color = 'text-orange-800';
+    type = 'CONFERENCE';
+  } else if (title.includes('workshop')) {
+    icon = <Webinar />;
+    color = 'text-blue-400';
+    type = 'WORKSHOP';
+  }
+
+  const defaultCover = '/img/homepage/confBlurBg.png';
+  let active = true;
+  if(currentDate > post.date){
+    active = false
+  }
+  return (
+    <li key={id} className={`${className}`}>
+      <article className="h-full rounded-lg shadow-md hover:shadow-lg">
+          <a>
```

Comment:
This `<a>` tag is left blank without any href tag. Do you want to direct to the Github issue link, created for meeting, using this component?

Reply:
Yea, i forgot to add it. Thanks for pointing it out :)